packages/core/services/DatabaseService/index.ts

```diff
@@ -12,6 +12,13 @@ import SQLBuilder from "../../entity/SQLBuilder";
 export default abstract class DatabaseService {
     protected readonly SOURCE_METADATA_TABLE = "source_metadata";
     public static readonly LIST_DELIMITER = ",";
+    // THIS CAN NEVER CHANGE, THIS VALUE IS INCLUDED IN THE EMT DATA RELEASE
```

Comment:
Maybe expand this with a link to the data release location, and the date?  I'm just thinking the new developer looking at this in 2027, with no context is going to be very confused.   Also, with no context, 'This' is unclear.  
maybe: "Link as a datatype must be hardcoded, and can not change without breaking visibility in the dataset released in 2024 as part of the paper at http://some.s3.url.dataset"

Hmm.  Could we make a release tag of BFF that goes with the paper?  That might make sense if BFF is going to be eternally tied to the paper release.

Reply:
> Maybe expand this with a link to the data release location, and the date?

Definately, good idea

> Could we make a release tag of BFF that goes with the paper? 

No, but we could include a Git hash reference... unsure anyone would be able to use that in an a helpful way though.

Reply:
Also, we could certainly add other string names that mean "OPEN_FILE_LINK_TYPE" in the future, if data creators dislike that version for whatever reason.

Reply:
I updated the language in the comment as suggested, but I can't include a link to the paper until it is published so I made a note for myself to add that in later.